packages/web3-eth-personal/test/integration/personal.test.ts

```diff
@@ -106,6 +107,8 @@ describe('personal integration tests', () => {
 		const accountList = await ethPersonal.getAccounts();
 		// create a new account
 		const account = await ethPersonal.newAccount('cde');
+		// we need this for confirm transaction
```

Comment:
which transaction exactly that is here needs to be confimed?

Reply:
yes. you are right. removed